docs/contributing/overview.md

```diff
@@ -1 +1,53 @@
-This section goes deep explaning Tracee's internals, and how to build Tracee on different platforms for the users interested in contributing.
+## Contributing
```

Comment:
Once merged, you should maybe link to this file from the PR/issue templates? cc @grantseltzer

Reply:
Will do!